docs/kubernetes/user-guides/driver-install.md

```diff
@@ -20,4 +20,13 @@ $ ./deploy/setup-project.sh
 $ GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to get the service account key
 $ GCE_PD_DRIVER_VERSION=stable                    # Driver version to deploy
 $ ./deploy/kubernetes/deploy-driver.sh
-```
\ No newline at end of file
+```
+
+## Running the controller separately
+
+Traditionally, you run the CSI controllers with the GCE PD driver in the same Kubernetes cluster.
+Though, in some scenarios you might want to run the CSI controllers (csi-provisioner, csi-attacher, etc.) together with the GCE PD controller service of this driver separately from the Kubernetes cluster it serves (while the GCE PD driver with an activated node service still runs inside the cluster).
```

Comment:
Though this is one use case I don't think it's the only one. I would make the description more general: "there may be cases where you will only want to run a subset of the available driver services:
- brief description of controller srevice and its dependencies (for example GCP Service Account Creds)
- brief description of node service and dependencies (for example metadata server)"

You could take the scenario written here and make it an example following the description.